deegree-ogcapi-features/src/main/java/org/deegree/services/oaf/resource/ApiVersionResponseFilter.java

```diff
@@ -0,0 +1,44 @@
+package org.deegree.services.oaf.resource;
```

Comment:
Please add to `org.deegree.services.oaf.filter` instead, then it is consistent with the other added filters.

Reply:
Done.